player/videojs.coffee

```diff
@@ -96,12 +96,21 @@ window.VideoJSPlayer = class VideoJSPlayer extends Player
             if data.meta.textTracks
                 data.meta.textTracks.forEach((track) ->
                     label = track.name
-                    $('<track/>').attr(
-                        src: track.url
-                        kind: 'subtitles'
-                        type: track.type
-                        label: label
-                    ).appendTo(video)
+                    if track.default? and track.default
+                        $('<track/>').attr(
+                            src: track.url
+                            kind: 'subtitles'
+                            type: track.type
+                            label: label
+                            default: ''
```

Comment:
How did you test this?  The PR title implies you want the video player to enable subtitles, but my reading of the [relevant MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTrackElement) is that this attribute merely specifies that among multiple track options, this one should be the default selected, and doesn't necessarily control whether the track ought to be enabled or not (especially since CyTube is using a videojs plugin for the track selector -- one would probably need to consult the docs of that particular plugin to know for certain).

Regardless, I feel that designation of a default track should be independent of the user's choice to enable text tracks or not.  One clean way to do this without introducing more UI clutter would be to do what YouTube does: when the user turns captions on, remember that the next time a YouTube video plays, and similarly when they turn it off.

Something worth considering as well is whether the manifest format ought to have a field on the text track for the language code (ISO or IETF standard) and let the user specify their preferred subtitle language instead of relying on the manifest author to choose an appropriate default.